hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/snapshot/OMSnapshotPurgeRequest.java

```diff
@@ -101,15 +99,13 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager, TermIn
         }
 
         SnapshotInfo nextSnapshot =
-            SnapshotUtils.getNextActiveSnapshot(fromSnapshot, snapshotChainManager, omSnapshotManager);
+            SnapshotUtils.getNextActiveSnapshot(fromSnapshot, snapshotChainManager, ozoneManager);
```

Comment:
ozoneManager or omSnapshotManager ?

Reply:
`ozoneManager` is correct. We had an unnecessary function [OmSnapshotManager#getSnapshotInfo](
https://github.com/apache/ozone/pull/7112/files#diff-857b24769ac9c1c67d015bdffc1227dc2b4751de3790b9fd89269a3c46a5218aL688-L690). It is doing nothing other than getting `OzoneManager` and passing it to `SnapshotUtils`. We can directly pass `OzoneManager` to it.